watson-developer-cloud/discovery-components
# feat: allow displaying multiple passages from each document

#### What do these changes do/fix?

- refactor Result component to handle multiple passages
- stack passage and table results vertically
- adjust styling accordingly
- update tests / Storybook

#### How do you test/verify these changes?

- View search components in Storybook and make sure they look and behave correctly. Note that the SearchResults story now has a result with two passages.
- Link to the corresponding `discovery-tooling` PR and make sure search results look correct when following the instructions in that PR.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
